src/processTargets/modifiers/ConditionalModifierStages.ts

```diff
@@ -0,0 +1,80 @@
+import { Target } from "../../typings/target.types";
+import {
+  Modifier,
+  ModifyIfUntypedModifier,
+} from "../../typings/targetDescriptor.types";
+import { ProcessedTargetsContext } from "../../typings/Types";
+import getModifierStage from "../getModifierStage";
+import { ModifierStage } from "../PipelineStages.types";
+
+abstract class ConditionalModifierBaseStage implements ModifierStage {
+  private nestedStage_?: ModifierStage;
+  protected suppressErrors = false;
```

Comment:
This feels orthogonal to conditional modifier.  Should really just use #1015 once we have that.  I'm ok with leaving this as is for now, as that machinery isn't there yet, but prob worth adding a checkbox to #1015 to use that machinery here